docs/european-user-consent.mdx

```diff
@@ -18,6 +18,16 @@ ad requests to Google serve personalized ads, with ad selection based on the use
 
 ## Handling consent
 
+To setup and configure ads consent collection, first of all:
+
+- Enable and configure GDPR and IDFA messaging in the [Privacy & messaging section of AdMob's Web Console](https://apps.admob.com/v2/privacymessaging).
+
+- For Android, add the following rule into
+  `android/app/proguard-rules.pro`:
+  ```
+  -keep class com.google.android.gms.internal.consent_sdk.** { *; }
```

Comment:
I still can't believe this part is necessary, but as we analyzed it definitely is, so this doc pointer is fantastic thank you